### PR TITLE
Added new Database Language. Relational Database PostgreSQL

### PR DESCRIPTION
**Pull request by sanzous**
PostgreSQL with link to website, 2 tutorial videos for beginners, and 2 build-along projects, 1 project using node.js and PostgreSQL the other using python PostgreSQL and Heroku for hosting.

This pull request is in regards to Issue #56

### CODE REVIEW

File: src/components/backendPageContent/Database.js [hunk 1]
```diff
@@ -103,6 +103,50 @@ const Database = () => {
...
+                </a>
+                <ResourceBadge>Video</ResourceBadge>
+              </ResourceLink>
+              <ResourceLink>
+                <a
+                  href="https://www.youtube.com/playlist?list=PLillGF-RfqbaEmlPcX5e_ejaK7Y5MydkW"
+                  target="_blank">
+                  Recipe App Using Node.js and PostgreSQL ~ Traversy Media
+                </a>
+                <ResourceBadge>Video</ResourceBadge>
```
Comment (Timonwa) [inline src/components/backendPageContent/Database.js:137]: Please change this badge to Playlist since it's a playlist

**Comment by Timonwa** [comment]
Also add the link to postgresql to the side bar.

**Comment by Timonwa** [comment]
Hi @sanzous . Thank you for your contribution. I have left two comments above, do check them and make the necessary changes b4 your pull request gets merged. Thank you. Please star ⭐ our repo if you haven't already. 🥰

**Comment by sanzous** [comment]
Made the changes! The video tag is now a playlist tag and it has been added to the sidebar.

**Comment by Timonwa** [comment]
@all-contributors please add @sanzous  for  data, code and content

**Pull request merged**